Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -208,48 +208,22 @@ import BraintreeDataCollector
             notifyCancel(completion: completion)
             return
         }
-
-        let clientDictionary: [String: String] = [
-            "platform": "iOS",
-            "product_name": "PayPal",
-            "paypal_sdk_version": "version"
-        ]
-
-        let responseDictionary: [String: String] = ["webURL": url.absoluteString]
-
-        var account: [String: Any] = [
-            "client": clientDictionary,
-            "response": responseDictionary,
-            "response_type": "web"
-        ]
-
-        if paymentType == .checkout {
-            account["options"] = ["validate": false]
-            if let request = payPalRequest as? BTPayPalCheckoutRequest {
-                account["intent"] = request.intent.stringValue
-            }
-        }
         
-        if let clientMetadataID {
-            account["correlation_id"] = clientMetadataID
-        }
-
-        var parameters: [String: Any] = ["paypal_account": account]
-        
-        if let payPalRequest, let merchantAccountID = payPalRequest.merchantAccountID {
-            parameters["merchant_account_id"] = merchantAccountID
+        guard let request = payPalRequest else {
```

Comment:
We can use the shorthand:
```suggestion
        guard let payPalRequest else {
```
And update below to `request: payPalRequest` in the `PayPalAccountPOSTEncodable`